setup.py

```diff
@@ -44,7 +44,7 @@ def version():
     install_requires=[
         "requests >= 2.28",
         "urllib3 >= 1.26",
-        "aiohttp >= 3.8.3",
+        "aiohttp >= 3.9.0b0",
```

Comment:
Why do we need a beta version?

Reply:
Aiohttp didn't run on latest version of python in CI. According to a stackoverflow post python 3.11 and/or 3.12 support is still in beta. Mostly wanted to try out if this was true. It seemed to fix the problem, but it broke CI on python 3.7 in exchange for that

Reply:
should have closed the PR afterwards

Reply:
Fun lol. Ok no worries